index.html

```diff
@@ -1,2 +1,3 @@
 feature 1
-Second feature
\ No newline at end of file
+Second feature
+Fourth feature
```

Comment:
Grym feature. Hur kom du på detta.

Reply:
Det bara dök upp i huvudet!

Reply:
Aha för jag brukade göra på det här sättet 

```suggestion
fourth feature
```
Med liten bokstav i början. Men Stor boksta är mycket bättre. Bra jobbat jag godkänner :)